packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -669,30 +670,46 @@ const AccountHeader = React.memo(
                   />
                 </InitialFocus>
               ) : isNameEditable ? (
-                <Button
-                  bare
+                <View
                   style={{
-                    fontSize: 25,
-                    fontWeight: 500,
-                    marginLeft: -5,
-                    marginTop: -5,
-                    backgroundColor: 'transparent',
-                    '& svg': { display: 'none' },
-                    '&:hover svg': { display: 'unset' }
+                    flexDirection: 'row',
+                    alignItems: 'center',
+                    gap: 3,
+                    '& .hover-visible': {
+                      opacity: 0,
+                      transition: 'opacity .25s'
+                    },
+                    '&:hover .hover-visible': {
+                      opacity: 1
+                    }
                   }}
-                  onClick={() => onExposeName(true)}
                 >
-                  {accountName}
-
-                  <Pencil1
+                  <View
                     style={{
-                      width: 11,
-                      height: 11,
-                      marginLeft: 5,
-                      color: colors.n4
+                      fontSize: 25,
+                      fontWeight: 500,
+                      marginRight: 5,
+                      marginBottom: 5
                     }}
-                  />
-                </Button>
+                  >
+                    {accountName}
+                  </View>
+
+                  <NotesButton id={`account-notes-${account.id}`} />
```

Comment:
Small nit: this could just be `account-${account.id}`; the "notes" is redundant as a key for these notes

Reply:
Sure. Done here: https://github.com/actualbudget/actual/pull/385/commits/6ef5253d38067ce41a3e2a6eadd9381fd63e662f